DDDEastAnglia.Tests/Helpers/Email/SendGrid/HtmlRendererFacts.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DDDEastAnglia.Helpers.Email.SendGrid;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Helpers.Email.SendGrid
+{
+    [TestFixture]
+    public class HtmlRendererFacts
+    {
+        [TestFixture]
+        public class Render_Should
+        {
+            [Test]
+            public void Convert_Markdown_To_HTML()
+            {
+                var sut = CreateSut();
```

Comment:
I can see where you're coming from, but I generally find that I don't actually care about the specifics of the SUT in a test, what I care about is understanding what is fulfilling that role in the test, the same way I care about the expected and actual results. Here, all I really care about is obtaining an instance of the SUT.